File: app/src/main/java/com/udacity/stockhawk/StockHawkApp.java
package com.udacity.stockhawk;

import android.app.Application;

import com.jakewharton.threetenabp.AndroidThreeTen;

import timber.log.Timber;

public class StockHawkApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        AndroidThreeTen.init(this);

        if (BuildConfig.DEBUG) {
            Timber.uprootAll();
            Timber.plant(new Timber.DebugTree());
        }
    }
}

File: app/src/main/java/com/udacity/stockhawk/StockHawkApp.kt
package com.udacity.stockhawk

import android.app.Application

import com.jakewharton.threetenabp.AndroidThreeTen

import timber.log.Timber

class StockHawkApp : Application() {

    override fun onCreate() {
        super.onCreate()
        AndroidThreeTen.init(this)

        if (BuildConfig.DEBUG) {
            Timber.uprootAll()
            Timber.plant(Timber.DebugTree())
        }
    }
}
